lib/ansible/utils/hashing.py

```diff
@@ -93,3 +93,6 @@ def md5(filename):
         raise ValueError('MD5 not available.  Possibly running in FIPS mode')
     return secure_hash(filename, _md5)
 
+from binascii import crc32 as crc32
+def crc32(data):
```

Comment:
the imported 'crc32' name and this method name collide.

filter/core.py expects this method to be called 'crc32_s' which doesn't exist.
I think you meant to name this crc32_s  (or better, crc32s to match existing md5s).

At top of file:
``` python
import binascii
```
(before the 'import os' since those are python std lib packages and 'binascii' is alphabetically first'

Then at end of module:

``` python
def crc32s(data):
     return binascii.crc32(data) & 0xffffffff
```